CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- **[Breaking]** MemPool: add `mmap` method
```

Comment:
I don't think adding a method is a breaking change :wink:

Reply:
I thought **[Breaking]** meant API breaking instead of application breaking, my bad :)